.github/workflows/update-i18n.yaml

```diff
@@ -20,7 +19,8 @@ jobs:
         aria2c https://raw.githubusercontent.com/SillyTavern/SillyTavern-i18n/main/generate.py
         aria2c https://raw.githubusercontent.com/SillyTavern/SillyTavern-i18n/main/requirements.txt
         pip install -r ./requirements.txt
-        python ./generate.py "" --sort-keys
+        python ./generate.py "" --sort-keys --auto-translate
```

Comment:
Remove auto-translate, they come out in bad quality.

Reply:
fixed with [`d181635` (#2382)](https://github.com/SillyTavern/SillyTavern/pull/2382/commits/d181635460791c249b28557ebbd08752101647ee) and [`169f8f0` (#2382)](https://github.com/SillyTavern/SillyTavern/pull/2382/commits/169f8f0eb05ff2250bd89118bd7248f069a0e864)